test/convenience/keyboard.test.ts

```diff
@@ -63,6 +72,83 @@ describe("Keyboard", () => {
         assertEquals(keyboard.resize_keyboard, false);
         assertEquals(keyboard.input_field_placeholder, "placeholder");
     });
+
+    it("can be transposed", () => {
+        function t(btns: string[][], expected: string[][]) {
+            assertEquals(
+                Keyboard.from(btns).toTransposed(),
+                Keyboard.from(expected),
+            );
+        }
+        t([["a"]], [["a"]]);
+        t([["a", "b", "c"]], [["a"], ["b"], ["c"]]);
+        t([["a", "b"], ["c", "d"], ["e"]], [["a", "c", "e"], ["b", "d"]]);
+        t(
+            [["a", "b"], ["c"], ["d", "e", "f"]],
+            [["a", "c", "d"], ["b", "e"], ["f"]],
+        );
+        const keyboard = Keyboard.from([["a", "b", "c"], ["d", "e"], ["f"]]);
+        assertEquals(keyboard.clone().toTransposed().toTransposed(), keyboard);
```

Comment:
Rename the method to `.transpose()`, or make it not mutate.

Reply:
It does not mutate

Reply:
We can remove the clone from the test. It's superfluous and confusing.

Reply:
Please do! And consider simplifying the name to `.transposed()`.

Reply:
@Loskir suggested that `toX` is consistent with JS, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted

Will adjust the test.